src/sidebars/search/search-controller.js

```diff
@@ -21,7 +21,12 @@
         }
 
         select(stock) {
-            this.selectionService.select(stock);
+            if (!this.store) {
+                this.store = window.storeService.open(window.name);
+            }
+            if (!this.store.isCompact() && stock.favourite) {
+                this.selectionService.select(stock);
+            }
```

Comment:
> So I can search for something, select it, then I go back to the regular view look at the chart.

Yeah, in compact view it's not possible, but in regular view it is. I see your point about bookmarks, but I don't like the fact that if you select something not in your favourites to look at, then select one of your favourites, you have no way of getting back to the first selection without going back and searching for it again. Unless we want to track recent searches or something, but that seems like more work than it's worth.